src/libraries/Microsoft.Extensions.Hosting.Abstractions/ref/Microsoft.Extensions.Hosting.Abstractions.cs

```diff
@@ -23,6 +23,7 @@ public abstract partial class BackgroundService : Microsoft.Extensions.Hosting.I
         public virtual System.Threading.Tasks.Task StartAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
         public virtual System.Threading.Tasks.Task StopAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+
```

Comment:
Can you revert these unnecessary changes in Microsoft.Extensions.Hosting.Abstractions? both this ref file and the src file below.